lib/svg-sprite/shape.js

```diff
@@ -708,6 +708,14 @@ SVGShape.prototype.setNamespace = function(ns) {
                 }
             });
 
+            // Substitute ID references in href attributes
```

Comment:
Please change this comment to what it really does

Reply:
Well, it is basically the same thing as the function above which is only for xlink:href. Which has the comment
`                // Substitute ID references in xlink:href attributes`